api/server/router/build/build.go

```diff
@@ -48,11 +48,14 @@ func (r *buildRouter) initRoutes() {
 // up to the client to choose which builder to use.
 func BuilderVersion(features map[string]bool) types.BuilderVersion {
 	// TODO(thaJeztah) move the default to daemon/config
+	bv := types.BuilderBuildKit
 	if runtime.GOOS == "windows" {
-		return types.BuilderV1
+		// BuildKit is not yet the default on Windows.
+		bv = types.BuilderV1
 	}
 
-	bv := types.BuilderBuildKit
+	// Allow the features field in the daemon config to override the
+	// default builder advertise.
 	if v, ok := features["buildkit"]; ok && !v {
 		bv = types.BuilderV1
```

Comment:
This won't do anything on Windows, because we already overrode it above 🙈 . I'll fix that.

Reply:
Fixed 😄